azure-pipelines.yml

```diff
@@ -12,6 +12,7 @@ trigger:
       - '*'
 
 variables:
+  ## TODO: create azure connection to AWS ECR
```

Comment:
remove this comment and create a ticket if you need to

Reply:
done